src/legacy/ui/public/visualize/loader/pipeline_helpers/build_pipeline.ts

```diff
@@ -333,8 +333,11 @@ const buildVisConfig: BuildVisConfigFunction = {
   },
 };
 
-export const buildVislibDimensions = (vis: any, timeRange?: any) => {
-  const schemas = getSchemas(vis, timeRange);
+export const buildVislibDimensions = async (
+  vis: any,
```

Comment:
I realize this is unchanged in the current PR, but we have a `Vis` type that's used in `getVisParams` which should probably also go here.